tmt/checks/watchdog.py

```diff
@@ -153,7 +153,7 @@ def do_ping(
         log = render_report_path(invocation)
 
         def _fail_parse_error(ping_output: str) -> None:
-            """ Handle unparseable ``ping`` output """
+            """ Handle unparsable ``ping`` output """
```

Comment:
hmm, can't say I'm a fan it correcting valid spelling. Not a big deal, but if it's somehow configurable, it would be cool if it could accept any valid spelling from British and US English, as long as it's not archaic.

Reply:
Unfortunately it only has [GB -> US](https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary_en-GB_to_en-US.txt), and that one is disabled right now